fv3core/stencils/dyn_core.py

```diff
@@ -116,7 +116,7 @@ def p_grad_c_stencil(
     """
     from __externals__ import hydrostatic
 
-    with computation(PARALLEL), interval(...):
+    with computation(PARALLEL), interval(0, -1):
```

Comment:
To make this change please also change in this file:
-        pgradc_domain = self.grid.domain_shape_compute(add=(1, 1, 0))
+        pgradc_domain = self.grid.domain_shape_compute(add=(1, 1, 1))